app/main/views/briefs.py

```diff
@@ -109,7 +109,11 @@ def get_brief(brief_id):
 
 @main.route('/briefs', methods=['GET'])
 def list_briefs():
-    briefs = Brief.query.order_by(Brief.published_at.desc(), Brief.id)
+    if request.args.get('human'):
+        briefs = Brief.query.order_by(Brief.status.desc(), Brief.published_at.desc(), Brief.id)
+    else:
+        briefs = Brief.query.order_by(Brief.published_at.desc(), Brief.id)
```

Comment:
I think non-human sort order would be only `Brief.id`, since publishing could move a brief between pages, reordering everything else